avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -369,6 +369,16 @@ param identityProfile object?
 @description('Optional. Enables Kubernetes Event-driven Autoscaling (KEDA).')
 param kedaAddon bool = false
 
+@description('Optional. Whether to enable VPA add-on in cluster. Default value is false.')
+param vpaAddon bool = false
```

Comment:
@PixelRobots Is this bool redundant? Can we just use the config of `vpaAddonAutoscaling` if a user wants to use this feature or not and set the `enabled` param in `verticalPodAutoscaler`? I am trying to avoid a situation where someone sets `vpaAddonAutoscaling` but does not set `vpaAddon` to true and wonders why the change didn't go through.